exporter/awsemfexporter/config.go

```diff
@@ -87,6 +87,13 @@ type Config struct {
 	// Otherwise, sending metrics as Embedded Metric Format version 0 (without "_aws")
 	Version string `mapstructure:"version"`
 
+	// StorageResolution is an OPTIONAL integer value representing the storage resolution for the corresponding metric. Setting this to 1 specifies this metric as
+	// a high-resolution metric, so that CloudWatch stores the metric with sub-minute resolution down to one second. Setting this to 60 specifies this metric as
+	// standard-resolution, which CloudWatch stores at 1-minute resolution. Values SHOULD be valid CloudWatch supported resolutions, 1 or 60. If a value is not
+	// provided, then a default value of 60 is assumed.
+	// https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html#CloudWatch_Embedded_Metric_Format_Specification_structure_metricdefinition
+	StorageResolution int `mapstructure:"storage_resolution"`
```

Comment:
Storage resolution should be able to be applied on a per metric basis imo. It should not be a global settings for all metrics processed by the exporter.